playbooks/init/evaluate_groups.yml

```diff
@@ -151,7 +139,6 @@
       ansible_become: "{{ g_sudo | default(omit) }}"
     with_items: "{{ g_new_node_hosts | default(g_node_hosts | default([], true), true) }}"
     changed_when: no
-    when: hostvars[item].openshift_node_bootstrap | default(True) | bool
```

Comment:
The group `oo_nodes_to_bootstrap` is now exactly the same as `oo_nodes_to_config`.  We do other things like add masters to this bootstrap group and also set up oo_exclude_bootstrapped_nodes.  This should be cleaned up to simplify the logic.